File: demo/src/main/java/com/example/MutualFundManager.java
package com.example;

public class MutualFundManager extends FinancialIntermediary {

    public MutualFundManager(String intermediaryName) {
        super(intermediaryName);
    }

    public String getIntermediaryName() {
        return intermediaryName;
    }

    @Override
    public void displayIntermediary() {
        System.out.println("Mutual Fund Manager: " + intermediaryName);
    }
}

File: src/MutualFundManager.cpp
#include "MutualFundManager.h"

MutualFundManager::MutualFundManager(const std::string& intermediaryName)
    : FinancialIntermediary(intermediaryName) {}

void MutualFundManager::displayIntermediary() const {
    std::cout << "Mutual Fund Manager: " << intermediaryName << std::endl;
}
